docs/IRISMapCubeSequence.rst

```diff
@@ -0,0 +1,212 @@
+.. _IRISMapCubeSequence:
+
+===================
+IRISMapCubeSequence
+===================
+
+This class provides access to IRIS Slit-jaw image (SJI) files as a list of
+IRISMapCube objects, where these sub-cubes are 2D or 3D described by their
+ own World Coordinate System (WCS).
+
+This class inherits from NDCubeSequence_, so many methods of IRISMapCubeSequence are linked
+to their description in NDCubeSequence_ documentation.
+
+Initialisation
+--------------
+
+The initialisation of an IRISMapCubeSequence is very similar to that of :ref:`IRISMapCube`,
+via the function ``read_iris_sji_level2_fits``. This function takes as argument a file name (string) or a list of filenames.
+
+Let us assume that we want to load an IRISMapCubeSequence object with two fits files called ``my_fits_file_0`` and ``my_fits_file_1``. We can create an IRISMapCubeSequence
+with as many fits files as necessary, but ideally with from the same observation (e.g. observations interrupted by eclipses). To load an IRISMapCubeSequence object as ``my_sequence``, we do: ::
+
+    >>> from irispy import read_iris_sji_level2_fits
+    >>> my_fits_files = [my_fits_file_0, my_fits_file_1]
+    >>> my_sequence = read_iris_sji_level2_fits(my_fits_files)
+
+As for IRISMapCube, if you don't have a lot of RAM memory or if you are loading a huge file,
```

Comment:
I think this should be ```As with IRISMapCube``` as ```As for``` implies we are switching the focus of the sentence back to ```IRISMapCube```, rather than just referencing it.